clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerUtils.java

```diff
@@ -221,4 +229,72 @@ else if (t instanceof KafkaException)
             throw new TimeoutException(e);
         }
     }
+
+    static void processRebalanceCallback(final EventHandler eventHandler,
```

Comment:
For my understanding, why do we place those methods in ConsumerUtils?

Reply:
Convenience for testing, mostly. Do you have a suggestion for a preferred location?